src/lib/network/clients/save_network.rb

```diff
@@ -99,7 +101,7 @@ def CopyConfiguredNetworkFiles
         # FIXME: this must be ripped out, refactored and tested
         # In particular, values containing slashes will break the last sed
         command = "\n" \
-          "source_file=#{source_file};dest_file=#{dest_file}\n" \
+          "source_file=#{source_file.shellescape};dest_file=#{dest_file.shellescape}\n" \
           "grep -v \"^[[:space:]]*#\" $source_file | grep = | while read option\n" \
```

Comment:
... but we're using it unquoted which will break. Use `\"$source_file\"` here, and quote *all* other shell variable expansions too ($key, $dest_file).

Proof:
```console
$ F="/tmp/one space"
$ touch "$F"
$ ls -l "$F"
-rw-r--r-- 1 mvidner users 0  7. pro 15.25 /tmp/one space
$ ruby -rshellwords -e 's = "/tmp/one space"; cmd = "f=#{s.shellescape}; ls -l $f; echo Retry; ls -l \"$f\""; system cmd'
ls: cannot access '/tmp/one': No such file or directory
ls: cannot access 'space': No such file or directory
Retry
-rw-r--r-- 1 mvidner users 0 Dec  7 15:25 '/tmp/one space'
```

Reply:
Given the FIXME above, it may be easier to rewrite this in plain Ruby.

Reply:
yep, I just worry it will get easily out of hands.

Reply:
I check what the code do and it in fact read and merge two sysconfig files. Maybe something for CFA, but it will be non-trivial change.

Reply:
OK, I address this in https://github.com/yast/yast-network/pull/709/commits/ac10b0988707aae95591d88ef70e1e22f6c69c16 which I hope is easy enough to read and should be much smarter then now.

Reply:
Great!